gandalf/src/main/java/io/github/btkelly/gandalf/utils/LoggerUtil.java

```diff
@@ -23,26 +23,35 @@
 public final class LoggerUtil {
 
     private static final String LOGGER_TAG = "Gandalf";
+    private static LogLevel level = LogLevel.NONE;
 
     public enum LogLevel {
```

Comment:
well we apply the @LogLevel annotation to the method parameter for type safety in this case